src/mnode/mnode-pastel.h

```diff
@@ -517,6 +522,8 @@ class CArtTradeTicket : public CPastelTicket
         READWRITE(m_nTimestamp);
         READWRITE(m_txid);
         READWRITE(m_nBlock);
+        READWRITE(nftRegTnxId);
```

Comment:
Do we need to increase version and do version checking like in CPastelIDRegTicket SerializationOp method?